pkg/tester/construction.go

```diff
@@ -402,7 +402,7 @@ func (t *ConstructionTester) waitForTip(ctx context.Context) (int64, error) {
 			return tipIndex, nil
 		}
 
-		log.Println("waiting for implementation to reach tip before testing...")
+		log.Printf("waiting for implementation to reach tip before testing... (current index: %d, tip delay: %d)\n", currentIndex, t.config.TipDelay)
```

Comment:
I am thinking if we want to show the current sync status and progress, other than tip delay probably we can also add current block timestamp so that we know how far are we from the tip